test/examples/15-drag-from-outside.jsx

```diff
@@ -88,7 +88,13 @@ class ShowcaseLayout extends React.Component {
         <button onClick={this.onCompactTypeChange}>
           Change Compaction Type
         </button>
-        <div className="droppable-element" draggable={true} unselectable="on">
+        <div
+          className="droppable-element"
+          draggable={true}
+          unselectable="on"
+          // this is a hack for firefox
+          onDragStart={e => e.dataTransfer.setData("text/plain", "")}
```

Comment:
?????

Reply:
@STRML I added description about this firefox behaviour